<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSystemSettings</class>
 <widget class="QDialog" name="DialogSystemSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="systemTab">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="path">
      <attribute name="title">
       <string>Path</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="5" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Temp</string>
         </property>
        </widget>
       </item>
       <item row="6" column="6">
        <widget class="QToolButton" name="path_fileDialog_script">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="path_lip"/>
       </item>
       <item row="6" column="1">
        <widget class="QToolButton" name="toolButton_4">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</iconset>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Database</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="path_database"/>
       </item>
       <item row="3" column="1">
        <widget class="QToolButton" name="toolButton_2">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</iconset>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="path_home_commentDialog">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</iconset>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="6">
        <widget class="QToolButton" name="path_fileDialog_temp">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QToolButton" name="toolButton_3">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</iconset>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QToolButton" name="toolButton_5">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</iconset>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="path_temp"/>
       </item>
       <item row="8" column="2">
        <widget class="QLineEdit" name="path_output"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Init</string>
         </property>
        </widget>
       </item>
       <item row="2" column="6">
        <widget class="QToolButton" name="path_fileDialog_database">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QToolButton" name="path_fileDialog_home">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QToolButton" name="toolButton_6">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</iconset>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="path_database_commentDialog">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</iconset>
         </property>
         <property name="popupMode">
          <enum>QToolButton::DelayedPopup</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="6">
        <widget class="QToolButton" name="path_fileDialog_init">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="path_home"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Lip</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLineEdit" name="path_init"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Output</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="path_script"/>
       </item>
       <item row="9" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Home</string>
         </property>
        </widget>
       </item>
       <item row="3" column="6">
        <widget class="QToolButton" name="path_fileDialog_lip">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Script</string>
         </property>
        </widget>
       </item>
       <item row="8" column="6">
        <widget class="QToolButton" name="path_fileDialog_output">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="database">
      <attribute name="title">
       <string>Database</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>In</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QToolButton" name="database_fileDialog_out">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="database_in"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Climate</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="database_out"/>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="database_fileDialog_in">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QToolButton" name="database_fileDialog_climate">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="database_climate"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Out</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="toolButton_15">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QToolButton" name="toolButton_16">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QToolButton" name="toolButton_17">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="settings">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="10" column="0">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Thread Count</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="settings_multithreading">
         <item>
          <property name="text">
           <string>true</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>false</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="settings_responsive">
         <item>
          <property name="text">
           <string>true</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>false</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="settings_threadCount">
         <property name="text">
          <string>-1</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="settings_logLevel">
         <item>
          <property name="text">
           <string>warning</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>error</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>disabled</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Log Level</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Enable Expression Linearization</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Multithreading</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="settings_expressionLinearizationEnabled">
         <item>
          <property name="text">
           <string>true</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>false</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Debug Output</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="settings_debugOutputAutoSave">
         <item>
          <property name="text">
           <string>true</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>false</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Responsive</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="settings_debugOutput">
         <item>
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>64</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>128</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>256</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="settings_randomSeed">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Debug Ouput Autosave</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Random Seed</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="javascript">
      <attribute name="title">
       <string>Javascript</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="2">
        <widget class="QLineEdit" name="javascript_fileName"/>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="javascript_fileDialog_fileName">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>File Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="toolButton_18">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</normaloff>../../../../../Downloads/abc_FILL0_wght400_GRAD0_opsz48.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox_systemSettingsDialog">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_systemSettingsDialog</sender>
   <signal>accepted()</signal>
   <receiver>DialogSystemSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_systemSettingsDialog</sender>
   <signal>rejected()</signal>
   <receiver>DialogSystemSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
